Description: Depend on the individual JEE spec jars instead of the big all in one artifact.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/quartz-core/pom.xml
+++ b/quartz-core/pom.xml
@@ -43,6 +43,18 @@
       <version>5.0-2</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-jta_1.2_spec</artifactId>
+      <version>debian</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
+      <version>3.1</version>
+      <scope>provided</scope>
+    </dependency>
 
     <!-- test dependencies -->
     <dependency>
--- a/quartz-jobs/pom.xml
+++ b/quartz-jobs/pom.xml
@@ -25,6 +25,18 @@
       <version>6.0-5</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-ejb_3.2_spec</artifactId>
+      <version>debian</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-jms_1.1_spec</artifactId>
+      <version>debian</version>
+      <scope>provided</scope>
+    </dependency>
     <dependency> 
       <groupId>javax.mail</groupId>
       <artifactId>mail</artifactId>
--- a/quartz-plugins/pom.xml
+++ b/quartz-plugins/pom.xml
@@ -34,6 +34,12 @@
       <version>5.0-2</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-jta_1.2_spec</artifactId>
+      <version>debian</version>
+      <scope>provided</scope>
+    </dependency>
     <!-- test dependencies -->
     <dependency>
       <groupId>org.slf4j</groupId>
